### DISCUSSION TITLE
Does the Dashboard pie graphic support more than 10 elements?

### DISCUSSION THREAD

**Discussion by Jacob-Tate** [Security-Onion-Solutions/securityonion#8202] [Unsupported Versions] [answered]
When using the table we have that ability to increase the number of elements we are fetching and we are able to control the number of elements showing on a single page of the table. When using the pie chart there doesn't appear to be any methods available to choose a discrete number or even increase the number of rendered elements.

For example when fetching 500 unique elements I can see the shares of the first 10 elements then the rest are put under the "other" category.

![image](https://user-images.githubusercontent.com/4615660/176068224-870d660d-a78c-4b5e-bd1e-d6c6c971add3.png)

**Comment by jertel** [comment, marked as answer]
The default limit of 10 was chosen to keep the clutter minimized on the chart and on the optional legend. As that number increases, the pie slices get smaller and the legend grows which makes gleaning information from the chart more challenging.

Technically yes, SOC can be provided a setting to override the default, but it is not currently exposed as a salt pillar. Therefore, changing this setting would make future SO upgrades more complicated for your installation.
